dbms/src/Columns/ColumnString.h

```diff
@@ -120,11 +120,16 @@ class ColumnString final : public IColumn
         }
         else
         {
-            const size_t old_size = chars.size();
-            const size_t size_to_append = src.offsets[0];
-            const size_t new_size = old_size + size_to_append;
-
-            chars.resize(new_size);
+            size_t size_to_append = 0;
+            size_t old_size = chars.size();
+            size_t new_size = old_size;
+            if (src.offsets.size() > 0)
```

Comment:
This method assumes that src column have nth element,
so `src.offsets` is not empty.

No need for check.

Reply:
When it was called from MySQLBlockInputStream::insertDefaultValue, array was empty, resulting in SEGV